code/datums/repositories/decls.dm

```diff
@@ -117,3 +117,7 @@ var/global/repository/decls/decls_repository = new
 	SHOULD_CALL_PARENT(FALSE)
 	PRINT_STACK_TRACE("Prevented attempt to delete a /decl instance: [log_info_line(src)]")
 	return QDEL_HINT_LETMELIVE
+
+/decl/Clone()
+	SHOULD_CALL_PARENT(FALSE)
+	return src //Return a reference, since we're a singleton
```

Comment:
It used to, but because we're deep copying lists, it very likely we'll hit /decl in them, 
I guess maybe adding a proc to check if something is clonable first, or if its a singleton would make more sense.

Reply:
Just leaving this here for making the convo easier to follow, but this was corrected, and now Clone is never allowed to be called on /decl.